tools/jbrowse2/jbrowse2.xml

```diff
@@ -0,0 +1,873 @@
+                                        </conditional>
+                                    </conditional>
+                                </conditional>
+                            </section>
+                        </conditional>
+                    </repeat>
+                </repeat>
+                <repeat name="track_groups">
+                    <param name="category" value="Realistic"/>
+                    <repeat name="data_tracks">
+                        <conditional name="data_format">
+                            <param name="data_format_select" value="gff"/>
+                            <conditional name="useuri">
+                                <param name="annotation" value="gff3/interpro.gff"/>
+                                <param name="insource" value="history"/>
+                            </conditional>
+                            <conditional name="match_part">
+                                <param name="match_part_select" value="false"/>
+                            </conditional>
+                            <section name="jbcolor_scale">
+                                <conditional name="color_score">
+                                    <param name="color_score_select" value="none"/>
+                                </conditional>
+                                <conditional name="color">
+                                    <param name="color_select" value="automatic"/>
+                                </conditional>
+                            </section>
+                        </conditional>
+                    </repeat>
+                    <repeat name="data_tracks">
+                        <conditional name="data_format">
+                            <param name="data_format_select" value="gff"/>
+                            <conditional name="useuri">
+                                <param name="annotation" value="gff3/2.gff"/>
+                                <param name="insource" value="history"/>
+                            </conditional>
+                            <conditional name="match_part">
+                                <param name="match_part_select" value="true"/>
+                                <param name="name" value="cDNA_match"/>
+                            </conditional>
+                            <section name="jbcolor_scale">
+                                <conditional name="color_score">
+                                    <param name="color_score_select" value="none"/>
+                                </conditional>
+                                <conditional name="color">
+                                    <param name="color_select" value="automatic"/>
+                                </conditional>
+                            </section>
+                        </conditional>
+                    </repeat>
+                </repeat>
+            </repeat>
+            <param name="uglyTestingHack" value="enabled"/>
+            <output name="output">
```

Comment:
Can we use the in_archive_member tests https://docs.galaxyproject.org/en/latest/dev/schema.html#assertions-for-archives-zip-tar to the the zip archive?

Reply:
> Can we use the in_archive_member tests 

Not without some other changes. 

Tests are all aimed at the galaxy.xml configuration intermediate document, and it is only copied if testing because it's hard to get at from the files_path in tests.

Tests are just about all old Helena code and the file is copied to output during a test for convenience:
```
#if str($uglyTestingHack) == "enabled":
       &&   cp '$trackxml' '$output'
    #end if
```
You could introspect the files_path contents, but there is _no zip archive normally_ - only available if zip_out is set so the change format can work

Reply:
I have included one example, I think with this example we can test everything that we need.

Reply:
Nice work - thanks!